### PR TITLE
Siamesex

### PR DESCRIPTION
**Pull request by Phancisth**
New Python idiom/idiomatic code snippets

### CODE REVIEW

File: elasticsearch-2.2.0/python-idioms/idiom-if-statement.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    prime_less_than_10 = num in (5,3,2,7)
+
+def i35():
+    char = input("Enter a character A-Z")
+    if char in ('A','E','I','O','U')
+        print("Input is an vowel")
+
+def i36():
+    book_info = ' The Three Musketeers: Alexandre Dumas'
+    formatted_book_info = book_info.strip().upper().replace(':', ' by')
```
[the remark this answers is not in the document]
Comment (Phancisth) [reply elasticsearch-2.2.0/python-idioms/idiom-if-statement.py]: I have changed the snippet to be as follow:
```suggestion
def i36():
        if itemListEmpty() == True: 
              return "List is empty"
```

File: elasticsearch-2.2.0/python-idioms/idiom-if-statement.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+def i37(setence):
+    output = sentence.capitalize().swapcase().replace('I\'m', 'I am')
+    output = output.replace('You\'re','You are').replace('can\'t','cannot')
+    return output
```
[the remark this answers is not in the document]
Comment (Phancisth) [reply elasticsearch-2.2.0/python-idioms/idiom-if-statement.py]: This snippet is changed to:
```
def i37():
    if name: 
         print(name)
        print(address)
         count++`
```

File: elasticsearch-2.2.0/python-idioms/idiom-if-statement.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cragkhit) [reply elasticsearch-2.2.0/python-idioms/idiom-if-statement.py]: Ok krub.

File: elasticsearch-2.2.0/python-idioms/idiom-if-statement.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cragkhit) [reply elasticsearch-2.2.0/python-idioms/idiom-if-statement.py]: Ok cool.

**Comment by cragkhit** [comment]
Oops. There're conflicts. Please resolve them so I can merge. @AGS48353 @Phancisth

**Pull request merged**